front/app/components/routing/Redirect.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { useEffect } from 'react';
+import clHistory from 'utils/cl-router/history';
+
+interface Props {
+  method: 'push' | 'replace';
+  path: string;
+}
+
+const Redirect = ({ method, path }: Props) => {
```

Comment:
Why have this as a component, and not just a custom hook? 
Also - can we avoid `// eslint-disable-next-line react-hooks/exhaustive-deps`